src/app/delegation_backend/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why a separate license file for this app; there's already a license file at the git root.

Reply:
Let's remove it, yep